### ISSUE TITLE
Charts

### ISSUE THREAD

**Issue by MAlexakis** [MAlexakis/WiFi-Indoor-Locationing#1]
![1. Training set](https://user-images.githubusercontent.com/42608658/44660289-7a691480-aa07-11e8-841d-8feec2cc13d6.png)
![2. Validation set](https://user-images.githubusercontent.com/42608658/44660290-7a691480-aa07-11e8-9be5-4d8f7811be0c.png)
![3. 3D Graph that shows the distribution of the floors](https://user-images.githubusercontent.com/42608658/44660291-7a691480-aa07-11e8-990d-8a8feafc960a.png)

**Comment by MAlexakis** [comment]
![WAPs signal strength distribution](https://user-images.githubusercontent.com/42608658/44661200-36c3da00-aa0a-11e8-8ea9-2984fdaf8e7e.png)

**Comment by MAlexakis** [comment]
![Satelite Capture](https://user-images.githubusercontent.com/42608658/44661384-cf5a5a00-aa0a-11e8-8f2f-655d284eb833.jpg)

**Comment by MAlexakis** [comment]
<img width="201" alt="screen shot 2018-08-27 at 15 22 55" src="https://user-images.githubusercontent.com/42608658/44662142-219c7a80-aa0d-11e8-9fc4-f8e3a4117692.png">

**Comment by MAlexakis** [comment]
<img width="246" alt="screen shot 2018-08-27 at 15 50 31" src="https://user-images.githubusercontent.com/42608658/44663482-f87de900-aa10-11e8-9dd9-c7e58b23558c.png">

**Comment by MAlexakis** [comment]
[http://localhost:21116/session/viewhtml3e8044b5d5e6/index.html](url)

**Comment by MAlexakis** [comment]
![radial error distribution](https://user-images.githubusercontent.com/42608658/44664278-11879980-aa13-11e8-838e-7eec7ba073ba.png)

**Comment by MAlexakis** [comment]
![comparison between validation data and predictions](https://user-images.githubusercontent.com/42608658/44664397-6b885f00-aa13-11e8-82ef-a88fd0c07e70.png)
